repose-aggregator/components/filters/header-translation-filter/src/main/scala/org/openrepose/filters/headertranslation/HeaderTranslationFilter.scala

```diff
@@ -65,22 +65,29 @@ class HeaderTranslationFilter @Inject()(configurationService: ConfigurationServi
 
       sourceHeaders foreach { sourceHeader =>
         val originalHeaderName = sourceHeader.getOriginalName
-        val originalHeaderValues = httpRequest.getHeadersScala(originalHeaderName)
+        val originalHeaderValues = if (sourceHeader.isSplittable)
+          httpRequest.getSplittableHeaderScala(originalHeaderName) else
+          httpRequest.getHeadersScala(originalHeaderName)
```

Comment:
I could be wrong, but I thought a style-check would complain about not having braces in that case making what should be a one-liner be like 5 lines.  Still, I was iffy on it to begin with, and if it wasn't obvious to you what that meant on first glance, it's worth changing.  I'll change it.